tests/test_remo.py

```diff
@@ -389,12 +389,9 @@ def test_get_items(self):
         req = HTTPServer.requests_http[-1]
         self.assertEqual(response, body)
         self.assertEqual(req.method, 'GET')
-        self.assertEqual(req.path, '/api/remo/v1/events/?page=1')
-        # Check request params
-        expected = {
-            'page': ['1']
-        }
-        self.assertDictEqual(req.querystring, expected)
+        self.assertTrue(req.path == '/api/remo/v1/events/?orderby=ASC&page=1'
```

Comment:
This is tricky! I'm not sure why `req.path` contains these parameters. We have been using it in several places and it doesn't include them.
For instance: https://github.com/chaoss/grimoirelab-perceval/blob/master/tests/test_bugzilla.py#L892